Documentation/compiling-seissol.rst

```diff
@@ -1,44 +1,96 @@
-.. _Compiling_and_running_seissol:
+.. _build_seissol:
 
-Compiling and running SeisSol
+Compiling and Running SeisSol
 -----------------------------
 
 Compiling SeisSol
 ~~~~~~~~~~~~~~~~~
-Get the latest version of SeisSol on git by cloning the whole repository
+Get the latest version of SeisSol on git by cloning the whole repository,
```

Comment:
Should we add something like: "Get the latest release (https://github.com/SeisSol/SeisSol/releases), by cloning the whole repository: `git clone --recursive --branch vX.Y.Z https://github.com/SeisSol/SeisSol.git`"

Reply:
I'll add it. Maybe also a note that the master branch may be unstable? (but contains the latest features)

Reply:
Yes, that's a good idea.